gslib/tests/test_json.py

```diff
@@ -0,0 +1,36 @@
+from gslib.tests import testcase
+from gslib.utils import json_six
+from six import PY3
+
+class TestJsonConversion(testcase.GsUtilUnitTestCase):
+  @classmethod
+  def setUpClass(cls):
+    if PY3:
+      cls.text_json = str('[{"integer": 1,"decimal": 3.14,"text": '
+                          '"testing, 1, 2"}]')
+      cls.binary_json = bytes(b'[{"integer": 1,"decimal": 3.14,"text": '
+                              b'"testing, 1, 2"}]')
+    else:
+      cls.text_json = unicode('[{"integer": 1,"decimal": 3.14,"text": '
+                              '"testing, 1, 2"}]')
+      cls.binary_json = str('[{"integer": 1,"decimal": 3.14,"text": '
+                          '"testing, 1, 2"}]')
+    cls.keys = {"integer", "decimal", "text"}
+    cls.json = [
+      {
+        "integer": 1,
+        "decimal": 3.14,
+        "text": "testing, 1, 2"
+      }
+    ]
```

Comment:
Need a blank line before a method def'n in a class.

Reply:
Formatting change submitted